core/unit_test/TestReducers_a.hpp

```diff
@@ -12,4 +12,8 @@ TEST(TEST_CATEGORY, reducers_int) {
   TestReducers<int, TEST_EXECSPACE>::execute_integer();
 }
 
+TEST(TEST_CATEGORY, reducers_unsigned_int) {
+  TestReducers<unsigned int, TEST_EXECSPACE>::execute_integer();
```

Comment:
```suggestion
#ifdef KOKKOS_ENABLE_OPENMPTARGET  // FIXME_OPENMPTARGET Failing with clang 17
  if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenMPTarget>)
    GTEST_SKIP() << "Known to fail with OpenMPTarget and clang 17";
#endif
TestReducers<unsigned int, TEST_EXECSPACE>::execute_integer();
```

Do we need to disable tests for `OpenMPTarget` backend?

Reply:
No we are removing the OpenMPTarget backend.  See #8696

Reply:
Oh, did not know that
Then, fine